exit/test_exit.c

```diff
@@ -0,0 +1,518 @@
+/*
+ * Phoenix-RTOS
+ *
+ *    POSIX.1-2017 standard library functions tests
+ *    HEADER:
+ *    - stdlib.h
+ *    - unistd.h
+ *    TESTED:
+ *    - exit()
+ *    - _exit() (_Exit equivalent)
+ *    - atexit()
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Adam Debek
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <stdlib.h>
+#include <unistd.h>
+#include <sys/wait.h>
+#include <signal.h>
+#include <fcntl.h>
+#include <string.h>
+#include <unity_fixture.h>
+#include "common.h"
+
+#define PATH       "/tmp/testFile"
+#define CREATEMODE 0666
+#define TEST_STR   "test123"
+
+static int fd;
+static int val = 0;
+
+
+/* SIGCHLD signal handler */
+static void handler(int signum)
```

Comment:
Please move this handler definition closer to SIGCHILD test as it is used only there

Reply:
I think we shouldn't place function definitions among testcase definitions.